components/notifications.js

```diff
@@ -320,13 +331,13 @@ function Votification ({ n }) {
             <ForwardedUsers />
           </>}
       </small>
-      <div>
+      <div className={styles.detailsOverlay}>
         {n.item.title
           ? <Item item={n.item} />
           : (
             <div className='pb-2'>
               <RootProvider root={n.item.root}>
-                <Comment item={n.item} noReply includeParent clickToContext />
+                <Comment href={href} as={as} item={n.item} noReply includeParent clickToContext />
```

Comment:
Curious. Why does this bleed into the comment/text component?

Reply:
When a comment contains links or youtube videos I want to still be able to click on those items while the box still contains the link but I think I have another solution whereby I can do this by adding some z-index to those child links specifically.

Reply:
sad I messed up the rebase :cry:

Reply:
revert and just merge. I squash 'em anyway.

Reply:
on a normal comment like this 
![image](https://github.com/stackernews/stacker.news/assets/108441023/6edfb0fd-b290-4736-a584-8ebc1951da75)
the higlighted area is the text and therefor is not a link and cannot be right-clicked from here without adding a linnkBox.

I am trying to see if I can add some zIndex to not need these links

Reply:
Right, I've run into this probelm before but It looks like playing with the z-index gets be to this problem.
![image](https://github.com/stackernews/stacker.news/assets/108441023/6b8f5919-bcb9-42d4-bdd2-fa12369a6366)

even after adding a higher z-index to the dropdown it persists so I think it might be a dead end with z-index alone

Reply:
I see. You need the linkbox to get click events but it can't if it's behind `Text`. Hmmm I wonder if there's a `pointer-events: none` thing that can be done.

Reply:
Good idea! that worked great, took a little bit of restoring pointers with `pointer-events: auto` on some of the children elements but its much cleaner.